lego/apps/meetings/models.py

```diff
@@ -189,3 +189,29 @@ class ReportChangelog(BasisModel):
 
     class Meta:
         ordering = ["-created_at"]
+
+
+class MeetingTemplate(BasisModel):
+    name = models.CharField(max_length=255, blank=False, null=False)
+    report = ContentField(blank=True, allow_images=True)
+    location = models.CharField(max_length=255, blank=True)
+    start_time = models.DateTimeField(blank=True, null=True)
+    end_time = models.DateTimeField(blank=True, null=True)
+    description = models.TextField(blank=True, default="")
+    mazemap_poi = models.PositiveIntegerField(null=True)
+    report_author = models.ForeignKey(
+        User,
+        blank=True,
+        null=True,
+        related_name="meetingstemplates_reports",
+        on_delete=models.SET_NULL,
+    )
+    invited_users = models.ManyToManyField(User)
+    invited_groups = models.ManyToManyField(AbakusGroup)
```

Comment:
This is just copy paste from the Meeting model right? Is it possible to make this more scalable? Perhaps make a basis model both this and Meeting can inherit from? Perhaps even do `class MeetingTemplate(Meeting)` but you probably don't want everything from the meeting model 🤷🏼‍♂️

Reply:
They are slightly different because the meeting needs additional fields on the invited_users many-to-many